Added some css framework and registration js

th3boe opened the pull request:
CSS framework has been added to some extent, and I have now made it so that I can register an account, but currently with no banner or avatar.

th3boe committed:
Added some css framework and registration js

CSS framework has been added to some extent, and I have now made it so that I can register an account, but currently with no banner or avatar.

th3boe committed:
added login, made some changes

changes: css, registration, login

Fermain reviewed (changes requested):
I love the look and feel of the theme, but there's missing requirements still:

- View list of posts
- View single post
- Delete single post
- Update single post
- Sort list of posts
- Filter list of posts

Let me know when you are done with work and we can discuss what needs to be done

th3boe committed:
Login and Storage

fixed registration and login issue while also adding storage file for maybe later use.

th3boe committed:
newsfeed and post implementation

th3boe committed:
Not search

messed up with the branch, but this is newsfeed.

th3boe committed:
Merge pull request #4 from th3boe/search

Search - messed up with the branch

th3boe committed:
sent in to starte fresh branch!

th3boe committed:
Merge pull request #5 from th3boe/search

not Search
